TestProjects/UniversalGraphicsTest/Assets/Scenes/110_URPShadersAlphaOutputCustomFeature/RenderTransparents.cs

```diff
@@ -0,0 +1,75 @@
+using UnityEngine;
```

Comment:
I wonder if you couldn't test this by adding the RenderObjects pass feature?

Reply:
Was able to. Killed that custom feature.